tests/schema/fields.py

```diff
@@ -0,0 +1,52 @@
+from django.db.models.fields.related import (
+    create_many_to_many_intermediary_model,
+    ManyToManyField, ManyToManyRel, RelatedField,
+    RECURSIVE_RELATIONSHIP_CONSTANT, ReverseManyRelatedObjectsDescriptor,)
```

Comment:
I prefer the bracket on the next line so you needn't move it when adding more items